server.php

```diff
@@ -137,10 +137,11 @@ function valet_default_site_path($config)
             // match dir for lowercase, because Nginx only tells us lowercase names
             if (strtolower($file) === $siteName) {
                 $valetSitePath = $path.'/'.$file;
-                break;
+                break 2;
             }
             if (strtolower($file) === $domain) {
                 $valetSitePath = $path.'/'.$file;
+                break 2;
             }
         }
         closedir($handle);
```

Comment:
While `break 2` is simple and expressive, that means the `closedir()` doesn't get called.
I haven't checked lately: does PHP always properly close all opened file handles at end of execution?

Reply:
You're right, I updated the code to break from the deepest loop when a match is found and break from the outside loop when `$valetSitePath` is not `null` anymore.